docs/documentation/quartz-3.x/how-tos/job-template.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Job Template
+---
+
+# Job Template
+
+This page tries to pull together a variety of common recommendations listed throughout the documentation
+into one page can be easily referenced.
+
+```csharp
+public class ASampleJob : IJob
+{
+    public static readonly JobKey Key = new JobKey("sample-job", "examples");
+
+    public async Task Execute(IJobExecutionContext context)
+    {
+        try 
+        {
+            // get data out of the MergedJobDataMap
+            var value = context.MergedJobDataMap.GetString("some-vaule");
+            
+            // ... do work
+        } catch (Exception ex) {
+            // do you want the job to refire?
+            throw new JobExecutionException(msg: "", refireImmediately: true, cause: ex);
```

Comment:
Yeah, I hear you.  Do you still consider this try/catch a best practice? If so I'll keep it, if not - I can take it out. We can then have a section that specific to `Exception Handling` because I def have some questions there.

Reply:
I think this is a good example to bring the handling known. I wonder if for v4 it should be more visible in the API or job definition as it's now really hard to discover.